less/about/common.less

```diff
@@ -20,3 +20,22 @@ html, body, #appContainer, #appContainer > div {
 body {
   font-size: 100%;
 }
+
+.linkText {
```

Comment:
Can you please use Aphrodite, so move this css into js file. More info here: https://github.com/brave/browser-laptop/wiki/Refactoring-styles-to-Aphrodite. Thank you :smiley: